tests/e2e/tests/regression/browser/stream-key.e2e.ts

```diff
@@ -0,0 +1,110 @@
+import { Chance } from 'chance';
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
+import { rte } from '../../../helpers/constants';
+import { BrowserPage, CliPage } from '../../../pageObjects';
+import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+
+const browserPage = new BrowserPage();
+const cliPage = new CliPage();
+const chance = new Chance();
+
+const field = chance.word({length: 5});
+const value = chance.word({length: 5});
+let keyName = chance.word({length: 20});
+
+fixture `Stream key`
+    .meta({
+        type: 'regression',
+        rte: rte.standalone
+    })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async() => {
+        await browserPage.deleteKeyByName(keyName);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    });
+test('Verify that user can see a Stream in a table format', async t => {
+    const streamFields = [
+        'Entry ID',
+        field
+    ];
+    keyName = chance.word({length: 20});
+    const command = `XADD ${keyName} * '${field}' '${value}'`;
+    //Add new Stream key
+    for(let i = 0; i < 5; i++){
+        await cliPage.sendCommandInCli(command);
+    }
+    //Open key details and check Steam format
+    await browserPage.openKeyDetails(keyName);
+    await t.expect(browserPage.virtualTableContainer.visible).ok('The Stream is displayed in a table format');
+    for(const field of streamFields){
+        await t.expect(browserPage.streamEntriesContainer.textContent).contains(field, 'The Stream fields are in the table');
+    }
+    await t.expect(browserPage.streamFieldsValues.textContent).contains(value, 'The Stream field value is in the table');
+});
+test('Verify that user can sort ASC/DESC by Entry ID', async t => {
+    keyName = chance.word({length: 20});
+    const command = `XADD ${keyName} * '${field}' '${value}'`;
+    //Add new Stream key
```

Comment:
Mention you do that 5 times. Also, (just suggestion) can be nice to create a function of sending command that cand do that multiple time instead of use this loop each time

Reply:
done